packages/server/lib/util/print-run.ts

```diff
@@ -369,6 +392,8 @@ export function renderSummaryTable (runUrl: string | undefined, results: any) {
       console.log(`  Recorded Run: ${formatPath(runUrl, undefined, 'gray')}`)
       console.log('')
     }
+
+    maybeLogCloudRecommendationMessage(runs, record)
```

Comment:
Nit: I feel like this function call doesn't belong in this function but rather right after the `renderSummaryTable`.

Reply:
Good call 👍🏻